brew.py

```diff
@@ -1,104 +1,126 @@
-import os, platform, subprocess, dotbot
-
+import os
+import platform
+import subprocess
+import dotbot
 class Brew(dotbot.Plugin):
     _brewDirective = "brew"
     _caskDirective = "cask"
     _tapDirective = "tap"
     _brewFileDirective = "brewfile"
+    _defaults = {}
+    _forceIntelOption = "force_intel"
 
     def can_handle(self, directive):
         return directive in (self._tapDirective, self._brewDirective, self._caskDirective, self._brewFileDirective)
 
     def handle(self, directive, data):
+        self._defaults = self._context.defaults().get(directive, {})
```

Comment:
Seems like here you wanted to get defaults for a given directive (e.g. `brewflie`, `tap`, etc.). But due to the fact that `self._defaults` has wider scope, it has a side effect of leaking these defaults to other directives.